package.json

```diff
@@ -218,6 +218,12 @@
     ]
   },
   "scripts": {
+    "start": "babel-node tools/run start",
+    "start:all": "yarn db:start && sleep 5 && yarn db:seed && yarn start",
```

Comment:
We might want to add `yarn db:stop` first so that start:all always works. What do you think?